Fri Sep 26 15:03:38 CEST 2014 : Apache Derby Network Server 10.10.1.3 - (1557168) wurde gestartet und ist bereit, Verbindungen auf Port 1527 zu akzeptieren.
----------------------------------------------------------------
Fri Sep 26 15:03:41 CEST 2014:
Booten von Derby-Version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168): Instanz a816c00e-0148-b20f-1a58-fffff4db26be
für Datenbankverzeichnis C:\wiki\Jahap\jahap\src\main\resources\DATABASE\JAHAPTEST mit Class Loader sun.misc.Launcher$AppClassLoader@1d44bcfa
Geladen aus file:/C:/wiki/jdk1.8.0_11/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_11-b12
user.dir=C:\wiki\jdk1.8.0_11\db
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=C:\wiki\Jahap\jahap\src\main\resources\DATABASE
Datenbank-Class Loader wurde gestartet - derby.database.classpath=''
Fri Sep 26 15:03:50 CEST 2014 Thread[DRDAConnThread_2,5,main] (XID = 56376), (SESSIONID = 0), (DATABASE = JAHAPTEST), (DRDAID = {1}), Cleanup action starting
java.sql.SQLNonTransientConnectionException: Verbindung konnte nicht authentifiziert werden. Grund: Ungültige Authentifizierung..
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.checkUserCredentials(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: java.sql.SQLException: Verbindung konnte nicht authentifiziert werden. Grund: Ungültige Authentifizierung..
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 19 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Verbindung konnte nicht authentifiziert werden. Grund: Ungültige Authentifizierung..
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.checkUserCredentials(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Fri Sep 26 15:03:50 CEST 2014 Thread[DRDAConnThread_2,5,main] (DATABASE = JAHAPTEST), (DRDAID = {1}), Verbindung konnte nicht authentifiziert werden. Grund: Ungültige Authentifizierung..
----------------------------------------------------------------
Fri Sep 26 15:03:57 CEST 2014:
Booten von Derby-Version The Apache Software Foundation - Apache Derby - 10.10.1.3 - (1557168): Instanz c2db804d-0148-b20f-1a58-fffff4db26be
für Datenbankverzeichnis C:\wiki\Jahap\jahap\src\main\resources\DATABASE\JAHAPs mit Class Loader sun.misc.Launcher$AppClassLoader@1d44bcfa
Geladen aus file:/C:/wiki/jdk1.8.0_11/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_11-b12
user.dir=C:\wiki\jdk1.8.0_11\db
os.name=Windows 7
os.arch=amd64
os.version=6.1
derby.system.home=C:\wiki\Jahap\jahap\src\main\resources\DATABASE
Datenbank-Class Loader wurde gestartet - derby.database.classpath=''
----------------------------------------------------------------
Fri Sep 26 17:43:23 CEST 2014: Derby-Engine wird heruntergefahren
